source/panel_predictor/__main__.py

```diff
@@ -59,9 +42,9 @@ def run_live(framework):
     args = vars(parser.parse_args())
 
     if args["mode"] == "video" and args["video"]:
-        run_video(video_path=args["video"], framework=args["framework"])
+        run_video(framework=args["framework"], capture=cv2.VideoCapture(args["video"]))
```

Comment:
`framework` and `capture` are required parameters. Why do you pass them as `framework=...`?

Reply:
It just specifies the order